static/src/questionnaires/QuestionnaireBodyCreate.vue

```diff
@@ -180,9 +208,38 @@ export default Vue.extend({
       const form = this.$refs.form
       return reportValidity(form)
     },
+    moveQuestionUp(themeIndex, qIndex) {
```

Comment:
I don't know if this is something like that you are doing https://jsfiddle.net/ygmosuk2/

But it could be at least refactor in a more general method that do up and down depending on the parameters passed.
Because, it is almost the same code in both method, up and down.